libs/langchain/langchain/chains/llm_bash/__init__.py

```diff
@@ -1 +1,8 @@
 """Chain that interprets a prompt and executes bash code to perform bash operations."""
+from langchain._api import warn_deprecated
+
+warn_deprecated(
```

Comment:
because we import LLMBash into `langchain/chains/__init__` this will always get raised

that seems less than ideal?

```
In [1]: from langchain.chains import LLMChain
/Users/harrisonchase/workplace/langchain/libs/langchain/langchain/chains/llm_bash/__init__.py:4: LangChainPendingDeprecationWarning: On 2023-10-09 the LLMBashChain will be moved to langchain-experimental
  warn_deprecated(
```

Reply:
Yeah good catch forgot about all behavior